sidebars.ts

```diff
@@ -925,20 +925,38 @@ module.exports = {
       collapsed: true,
       link: {type: 'doc', id: 'manage/manage-subscription/index'},
       items: [
-        'manage/manage-subscription/sumo-logic-flex-accounts',
         'manage/manage-subscription/sumo-logic-credits-accounts',
-        'manage/manage-subscription/upgrade-sumo-logic-credits-account',
+        'manage/manage-subscription/sumo-logic-flex-accounts',
+        'manage/manage-subscription/cloud-flex-legacy-accounts',
+        {
+          type: 'category',
+          label: 'Create and Manage Orgs',
+          collapsible: true,
+          collapsed: true,
+          link: {type: 'doc', id: 'manage/manage-subscription/create-and-manage-orgs/index'},
+          items: [
+            'manage/manage-subscription/create-and-manage-orgs/create-manage-orgs',
```

Comment:
The URL needs to be changed. Otherwise it will read as "manage/manage-subscription/create-and-manage-orgs/**create-manage-orgs/create-manage-orgs**".